README.md

```diff
@@ -2,38 +2,267 @@
+   mkdir "$env:AppData\notation\plugins\azure-kv"
+   Expand-Archive -Path notation-azure-kv_<version>_windows_amd64.zip -DestinationPath "$env:AppData\notation\plugins\azure-kv"
+   ```
+3. Try to run `notation plugin list` to show the installed plugin.
 
+## Getting started with a self-signed Azure Key Vault Certificate
+To demonstrate the basic use case, starting with a self-signed certificate is easier to understand how the Azure-kv plugin works with Notation, however, you **should not** use this in production because the self-signed certificate is not trusted.
+1. Install Azure CLI by following the [guide](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
+2. Login to Azure with Azure CLI, set the subscription and make sure the `GetCertificate` and `Sign` permission have been granted to your role:
+   ```sh
+   az login
+   az account set --subscription $subscriptionID
+   ```
+3. Create an Azure Key Vault and a self-signed certificate:
+   ```sh
+   resourceGroup=notationResource
+   keyVault=notationKV
+   location=westus
+   certName=notationSelfSignedCert
 
+   # create a resource group
+   az group create -n $resourceGroup -l $location
+   
+   # create a Azure Key Vault
+   az keyvault create -l $location -n $keyVault --resource-group $resourceGroup
 
+   # generate certificate policy
+   cat <<EOF > ./selfSignedPolicy.json
+   {
+     "issuerParameters": {
+       "certificateTransparency": null,
+       "name": "Self"
+     },
+     "keyProperties": {
+       "curve": null,
+       "exportable": false,
+       "keySize": 2048,
+       "keyType": "RSA",
+       "reuseKey": true
+     },
+     "secretProperties": {
+       "contentType": "application/x-pem-file"
+     },
+     "x509CertificateProperties": {
+        "ekus": [
+           "1.3.6.1.5.5.7.3.3"
+       ],
+       "keyUsage": [
+         "digitalSignature"
+       ],
+       "subject": "CN=Test Signer",
+       "validityInMonths": 12
+     }
+   }
+   EOF
+
+   # create self-signed certificate
+   az keyvault certificate create -n $certName --vault-name $keyVault -p @selfSignedPolicy.json
+
+   # get the key identifier
+   keyID=$(az keyvault certificate show -n $certName --vault-name $keyVault --query 'kid' -o tsv)
+   ```
+4. Prepare a container registry. You can follow the [guide](https://learn.microsoft.com/en-us/azure/container-registry/container-registry-get-started-portal?tabs=azure-cli) to create an Azure Container Registry which is recommended. 
+5. Sign the container image with Notation:
+   ```sh
+   # sign with azure-kv
+   notation sign localhost:5000/hello-world:v1 --id $keyID --plugin azure-kv
+   # example output
+   Warning: Always sign the artifact using digest(@sha256:...) rather than a tag(:v1) because tags are mutable and a tag reference can point to a different artifact than the one signed.
+   Successfully signed localhost:5000/hello-world@sha256:f54a58bc1aac5ea1a25d796ae155dc228b3f0e11d046ae276b39c4bf2f13d8c4
+   ```
```

Comment:
````suggestion
5. Sign the container image using `notation` and `notation-azure-kv`:
   ```sh
   notation sign localhost:5000/hello-world:v1 --id $keyID --plugin azure-kv
   ```
   Example output:
   ```
   Warning: Always sign the artifact using digest(@sha256:...) rather than a tag(:v1) because tags are mutable and a tag reference can point to a different artifact than the one signed.
   Successfully signed localhost:5000/hello-world@sha256:f54a58bc1aac5ea1a25d796ae155dc228b3f0e11d046ae276b39c4bf2f13d8c4
   ```
````

Reply:
Updated.